package.json

```diff
@@ -789,6 +789,7 @@
     "yargs": "^16.2.0"
   },
   "dependencies": {
+    "@types/node-fetch": "^2.6.2",
```

Comment:
Why the version different from devdependencies

Reply:
That is Type of so i think took latest by npm install